src/components/DateInput.jsx

```diff
@@ -217,7 +220,7 @@ class DateInput extends React.PureComponent {
             focused && styles.DateInput_input__focused,
             disabled && styles.DateInput_input__disabled,
           )}
-          aria-label={placeholder}
```

Comment:
Note: This is potentially not an appropriate label, if the placeholder is actually placeholder text and not a label. Arguably, this is a result of the design itself encouraging that behavior, but it's not ideal for accessibility.

Reply:
Yeah, I agree with that. Perhaps a better design, instead of this PR, is to just require users to pass in a value if they want an `aria-label` set to something, and not use the `placeholder` by default. Accessibility-wise, I think that would be more ideal. What do you think?

Reply:
Agreed - that way it's more explicit about what the label is. Worthy of a separate PR :)

Reply:
Opened it here #1594. It's a way better design than this PR anyways.